### PR TITLE
style: fix description item long text

### PR DESCRIPTION
**Pull request by hengkx**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   fix Descriptions item long text style.        |
| 🇨🇳 Chinese |  修复 Descriptions 长文本移除样式的问题。         |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/descriptions/style/index.less [hunk 1]
```diff
@@ -86,6 +86,7 @@
     color: @text-color;
     font-size: @font-size-base;
     line-height: @line-height-base;
+    word-break: break-all;
```
Comment (afc163) [inline components/descriptions/style/index.less:89]: break-all 不好，有时候英文单词不希望在行尾被切断。

**Comment by afc163** [comment]
image

File: components/descriptions/style/index.less (hunk 1 above)
Comment (hengkx) [reply components/descriptions/style/index.less:89]: 用break-word? 
image
image

Comment (afc163) [reply components/descriptions/style/index.less:89]: 嗯，break-word 好一点。

Comment (afc163) [reply components/descriptions/style/index.less:89]: `overflow-wrap: break-word;`

**Comment by afc163** [comment]
> fix description item long text style
> 修复description长文本样式

1. 补上标点。
2. 中英文之间加空格。
3. 使用正确的组件名 Descriptions。
4. 适当补充细节，使其表述明确：`修复 ... 长文本溢出的样式问题。`。

**Comment by hengkx** [comment]
image
这是什么问题？

**Pull request merged**